<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>782</width>
    <height>636</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>100</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Kniffel</string>
  </property>
  <property name="accessibleName">
   <string/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <property name="spacing">
    <number>20</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="WuerfelWidget" name="widget" native="true"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <property name="leftMargin">
        <number>30</number>
       </property>
       <item>
        <widget class="QCheckBox" name="checkBoxWuerfel1">
         <property name="focusPolicy">
          <enum>Qt::WheelFocus</enum>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxWuerfel2">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxWuerfel3">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxWuerfel4">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxWuerfel5">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Wuerfeln">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>Würfeln</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButton_Einser">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>Einser</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="pushButton_Zweier">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>Zweier</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="pushButton_Dreier">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>Dreier</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pushButton_Vierer">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>Vierer</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="pushButton_Fuenfer">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>Fünfer</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="pushButton_Sechser">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>Sechser</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="pushButton_Dreierpasch">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>Dreierpasch</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="pushButton_Viererpasch">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>Viererpasch</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="pushButton_FullHouse">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>Full House</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="pushButton_KleineStrasse">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>Kleine Straße</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="pushButton_GrosseStrasse">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>Große Straße</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QPushButton" name="pushButton_Kniffel">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>Kniffel</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QPushButton" name="pushButton_Chance">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>Chance</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_playerName_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>20</height>
          </size>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>Spieler an der Reihe:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_playerNamePrinted">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_roundCounter">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>Runde:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="label_4">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_rollCounter">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>Anzahl Würfe:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="label_rollCountePrinted">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_possibilities">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>250</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_registered">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>250</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_totalScore">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>100</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignHCenter|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_status">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_playerName">
         <property name="minimumSize">
          <size>
           <width>35</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>35</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>Name:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textEdit_PlayerName">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_acceptName">
         <property name="minimumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>Ok</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WuerfelWidget</class>
   <extends>QWidget</extends>
   <header>wuerfelwidget.h</header>
   <container>1</container>
   <slots>
    <signal>showPossibilities(QString)</signal>
    <signal>showRegistered(QString)</signal>
    <signal>showPlayerName(QString)</signal>
    <signal>showRoundCounter(QString)</signal>
    <signal>showRollCounter(QString)</signal>
    <signal>showError(QString)</signal>
    <signal>showStatus(QString)</signal>
    <signal>showScore(QString)</signal>
    <slot>wuerfeln()</slot>
    <slot>registerEinser()</slot>
    <slot>registerZweier()</slot>
    <slot>registerDreier()</slot>
    <slot>registerVierer()</slot>
    <slot>registerFuenfer()</slot>
    <slot>registerSechser()</slot>
    <slot>registerKleineStrasse()</slot>
    <slot>registerGrosseStrasse()</slot>
    <slot>registerDreierpasch()</slot>
    <slot>registerViererpasch()</slot>
    <slot>registerFullHouse()</slot>
    <slot>registerKniffel()</slot>
    <slot>registerChance()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>widget</sender>
   <signal>showRegistered(QString)</signal>
   <receiver>label_registered</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>568</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>widget</sender>
   <signal>showStatus(QString)</signal>
   <receiver>label_status</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>214</x>
     <y>136</y>
    </hint>
    <hint type="destinationlabel">
     <x>437</x>
     <y>493</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Wuerfeln</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>wuerfeln()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>48</x>
     <y>191</y>
    </hint>
    <hint type="destinationlabel">
     <x>251</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>widget</sender>
   <signal>showPlayerName(QString)</signal>
   <receiver>label_playerNamePrinted</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>255</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel">
     <x>534</x>
     <y>50</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>widget</sender>
   <signal>showRoundCounter(QString)</signal>
   <receiver>label_4</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>280</x>
     <y>62</y>
    </hint>
    <hint type="destinationlabel">
     <x>532</x>
     <y>68</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>widget</sender>
   <signal>showRollCounter(QString)</signal>
   <receiver>label_rollCountePrinted</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>279</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>527</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>widget</sender>
   <signal>showPossibilities(QString)</signal>
   <receiver>label_possibilities</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>179</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>481</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Einser</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>registerEinser()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>36</x>
     <y>234</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Zweier</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>registerZweier()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>159</x>
     <y>225</y>
    </hint>
    <hint type="destinationlabel">
     <x>206</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Dreier</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>registerDreier()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>272</y>
    </hint>
    <hint type="destinationlabel">
     <x>34</x>
     <y>273</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Vierer</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>registerVierer()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>193</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>288</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Fuenfer</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>registerFuenfer()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>57</x>
     <y>325</y>
    </hint>
    <hint type="destinationlabel">
     <x>25</x>
     <y>325</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Sechser</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>registerSechser()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>155</x>
     <y>325</y>
    </hint>
    <hint type="destinationlabel">
     <x>133</x>
     <y>334</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Dreierpasch</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>registerDreierpasch()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>66</x>
     <y>384</y>
    </hint>
    <hint type="destinationlabel">
     <x>28</x>
     <y>379</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Viererpasch</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>registerViererpasch()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>186</x>
     <y>379</y>
    </hint>
    <hint type="destinationlabel">
     <x>183</x>
     <y>386</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_FullHouse</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>registerFullHouse()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>55</x>
     <y>430</y>
    </hint>
    <hint type="destinationlabel">
     <x>31</x>
     <y>428</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_KleineStrasse</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>registerKleineStrasse()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>160</x>
     <y>434</y>
    </hint>
    <hint type="destinationlabel">
     <x>156</x>
     <y>447</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_GrosseStrasse</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>registerGrosseStrasse()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>45</x>
     <y>488</y>
    </hint>
    <hint type="destinationlabel">
     <x>16</x>
     <y>490</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Kniffel</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>registerKniffel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>133</x>
     <y>496</y>
    </hint>
    <hint type="destinationlabel">
     <x>133</x>
     <y>505</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Chance</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>registerChance()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>270</x>
     <y>493</y>
    </hint>
    <hint type="destinationlabel">
     <x>264</x>
     <y>507</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Wuerfeln</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>startGame()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>85</x>
     <y>188</y>
    </hint>
    <hint type="destinationlabel">
     <x>133</x>
     <y>181</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_acceptName</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>acceptPlayerName()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>585</x>
     <y>545</y>
    </hint>
    <hint type="destinationlabel">
     <x>497</x>
     <y>427</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>widget</sender>
   <signal>showScore(QString)</signal>
   <receiver>label_totalScore</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>126</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>417</x>
     <y>413</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>chosenDice(QString)</signal>
  <signal>showPlayerName(QString)</signal>
  <signal>showRoundCounter(QString)</signal>
  <signal>showRollCounter(QString)</signal>
  <signal>showPossibilities(QString)</signal>
  <slot>checkChosenDice()</slot>
  <slot>wuerfeln()</slot>
  <slot>register(Einser)</slot>
  <slot>registerEinser()</slot>
  <slot>registerZweier()</slot>
  <slot>registerDreier()</slot>
  <slot>registerVierer()</slot>
  <slot>registerFuenfer()</slot>
  <slot>registerSechser()</slot>
  <slot>registerDreierpasch()</slot>
  <slot>registerViererpasch()</slot>
  <slot>registerFullHouse()</slot>
  <slot>registerKleineStrasse()</slot>
  <slot>registerGrosseStrasse()</slot>
  <slot>registerKniffel()</slot>
  <slot>registerChance()</slot>
  <slot>startGame()</slot>
  <slot>acceptPlayerName()</slot>
 </slots>
</ui>
